### PR TITLE
inkscape: update to 1.0.2.

### PR DESCRIPTION
**Pull request by hervyqa**
Screenshot_20210224_225917

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)

- [x] I built this PR locally for my native architecture, (x86_64-musl)
- [ ] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [ ] aarch64-musl
  - [ ] armv7l
  - [ ] armv6l-musl

### CODE REVIEW

File: srcpkgs/inkscape/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'inkscape'
 pkgname=inkscape
-version=1.0.1
-revision=1
+version=1.0.2
+revision=2
```
Comment (ericonr) [inline srcpkgs/inkscape/template:4]: This should be left at 1

**Review by ericonr — COMMENTED** [review]
Also please remove the entire `case` statement in `pre_configure` and rename the function to `post_patch`.

**Comment by ericonr** [comment]
Also, please try adding `checkdepends="gtest-devel"`

**Comment by ericonr** [comment]
And please remove the `poppler-072.patch` file.

**Comment by ericonr** [comment]
I realise I'm asking a bunch, so I will try to fix all these locally and push, all right? The commit will still have credit to you for the update!

**Comment by hervyqa** [comment]
> I realise I'm asking a bunch, so I will try to fix all these locally and push, all right? The commit will still have credit to you for the update!

@ericonr this is what I want. moreover, I can't wait for this new release of Inkscape. so I need to close this pull request?

**Pull request closed without merging**